src/index.js

```diff
@@ -7,16 +7,26 @@ function select(id) {
 }
 
 // we may expose this eventually
-function queryDivByTestId(div, id) {
+function queryByTestId(div, id) {
   return div.querySelector(select(id))
 }
 
+// we may expose this eventually
+function getByTestId(div, id) {
+  const el = queryByTestId(div, id)
+  if (!el) {
+    throw new Error(`Unable to find element by ${select(id)}`)
```

Comment:
@kentcdodds Oops! If now `getByTestId` throws error, then `toBeInTheDOM` (which I'm working on it) wouldn't make sense right?

Reply:
Yeah, I'd thought of that. It still works for `queryByTestId` though.